internal/e2e-tests/main_test.go

```diff
@@ -337,13 +337,17 @@ func (s *E2ESuite) assertClusterCreation() {
 
 		s.click(page, `button:has-text("Save")`)
 
+		err = page.Locator("button#extensions-CP").WaitFor()
+
+		s.Require().NoError(err)
+
 		s.click(page, `button#extensions-CP`)
 
-		err = page.Locator(`span:has-text("qemu-guest-agent")`).WaitFor()
+		err = page.Locator(`span:has-text("usb-modem-drivers")`).WaitFor()
```

Comment:
changed extension name to avoid clashing with the integration tests, as I think we enable `qemu-guest-agent` there.